gulpfile.js

```diff
@@ -134,6 +144,10 @@ const javascripts = {
   }
 };
 
+const hashJavascriptFilenames = () => src(paths.dist + 'javascripts/*.js')
```

Comment:
Minor: this is inconsistent with the style we have for `scss` below, which has a `compile` task. Could we have made it a `hashFilenames` key in the existing `javascripts` const, rather than adding a new one?